### ISSUE TITLE
Do I need asus armory crate? 

### ISSUE THREAD

**Issue by hugopua** [ChromaControl/ChromaControl#59]
Hi. 

Do I need asus armory crate to install to run chroma connect? I plan to add aura terminal to my motherboard, will chroma connect work for razer synapse to control aura terminal? Thank you.

**Comment by AndrewBabbitt97** [comment]
Armory Crate is a requirement
No clue on Aura Terminal support, it may or may not work

Really why get Aura Terminal when Razer makes there own ARGB controller? https://www.razer.com/gaming-pc-accessories/razer-chroma-addressable-rgb-controller/RZ34-02140600-R3U1

**Comment by hugopua** [comment]
Thank you for your reply. I would like to build a fully aura sync pc and possible control by razer synapse too.

**Comment by nodiaque** [comment]
Armory Crate is NOT a requirement. I have a x570 tuf gaming and I only installed the aura service and it's working properly. No Aura Sync or Armory Crate.

**Comment by AndrewBabbitt97** [comment]
Let me rephrase: Armory  Create is a requirement to have support from me. The reason I say this is because you get HAL updates through Armory Crate. Yes lighting service is required for this to work at all, but I wont support anyone who hasn't ensured all device HALs are updated through Armory Crate along with installing the latest bios version.

**Comment by nodiaque** [comment]
Problem is you can't have Armory Crate and iCUE at the sametime.

But aside of that, you do know that armory crate actually install out of date drivers, firmware and components? I have a brand new computer I build a month ago, asus x570 tuf gaming pro. Installed armory crate and all the bells and whistle (that I ended up uninstalling one by one, cfosspeed turbo lan being the last one an hour ago switching my speed from 100mbps to 400mbps...) and I though Armory crate was a good piece. Until I read many place saying everything was outdated in it. I did like they say, go directly to asus support and oh, yeah, everything is outdated! Drivers, firmware, bios, even lighting service. Installing latest lighting service from Asus does install all HALL, it's part of the installation package. In fact, when you install armory crate, it download the lighting service.

**Comment by AndrewBabbitt97** [comment]
You can have Armory Crate and iCUE at the same time. You must remove the Aura plugin from iCUE first. (This will be documented on the upcoming website due in the next few weeks).

You cannot get the aura service installer from the driver download page for pretty much any new device from ASUS and even on the aura page only an armory crate download is available:

- [Aura download page](https://www.asus.com/campaign/aura/us/download.php)
- [Maximus XII Formula](https://rog.asus.com/motherboards/rog-maximus/rog-maximus-xii-formula-model/helpdesk_download)
- [Maximus XIII Extreme](https://rog.asus.com/us/motherboards/rog-maximus/rog-maximus-xiii-extreme-model/helpdesk_download)

Should be similar for all new ASUS boards. This is the direction ASUS is going and so this will be the only supported method of installing Lighting service along with any device HALs. I don't want the support nightmare of people trying to track down specific versions of lighting service and device HALs when they can be delivered from Armory Crate with little to no user intervention.

Using Armory Crate does not mean you are required to get driver updates from it as well, I get all my drivers from the support page. Armory Crate also provides support for other motherboard features such as the OLED screen on some higher end boards. The more configurations I have to support people using with Chroma Control the bigger the support burden. I am a single person working on this project, and so I cannot take on people using random versions of Lighting service. In many cases the support page also doesn't have the latest version of Aura available. ASUS tends to stop updating drivers on support pages for devices after a few years and I know we have people running some older hardware.

So in general, I support people using Armory Crate to get the latest version of Lighting service and device HALs. This is also the only way ASUS is delivering new versions of Lighting service and device HALs anymore for new devices. Anything outside of this configuration is not supported.

**Comment by nodiaque** [comment]
Oh I do agree that for support, saying "do you have the latest version" is the right way to go.

Problem with iCUE and aura is not the iCUE side. iCUE stop working when Aura is installed because Aura take control of everything and iCUE get a big red triangle. That's why I don't have any sync. It could work for 1-2 hours and then suddently, anything corsair related stop if controlled from iCUE. Even when removing the corsair plugin from aura, it still did some bug like that. Corsair say to not install both at the sametime.

**Comment by AndrewBabbitt97** [comment]
???? I'm confused as to what you are referring to.

There is no iCUE plugin for Aura, only the other way around. The only time this may become an issue is if you have Corsair RGB ram. Everything else should be fine.

**Comment by nodiaque** [comment]
That's exactly what I have, corsair vengeance pro rgb. Maybe a keyboard next but dont have it currently. Didn't knew its only the ram aura can do. I prefer icue effect anyway, specially with thermal sensing and all other option I can do with it.

The aura plugin for corsair is so useless since it doesn't implement argb header. Corsair non stop say the aura plugin/api/sdk doesn't allow for argb header, but hey, you do it.

**Comment by AndrewBabbitt97** [comment]
Original issue has been answered. If there's another issue open a new issue.

Answer to the original issue is Armory Crate is required, support will not be provided without it.

**Comment by gbcomins** [comment]
I have removed Armory Crate as every time I have it installed, it completely looses my discrete video card. Cannot find it through device manager or even Aida64 Engineer edition. The second I remove AC, it comes back to normal again. You can easily keep updated and current using the webpage and MyASUS. You say you will not support those without Armory Crate, bit naive considering , in my case, it is more problematic and useful

**Comment by AndrewBabbitt97** [comment]
Armory Crate provides AURA. Newer motherboards and GPUs use a newer version of Aura only provided by Armory Crate.

Device HAL updates are also only delivered through Armory Crate.

The old aura install page provides an old version of aura that does not get tested. You may use it but there will be no support on my side for anyone using it.

**Comment by gbcomins** [comment]
On further research, it will seem that you are completely correct. Forgive my ignorance. Unfortunately, my issue still stands but that will be for another forum. ty
